README.md

```diff
@@ -148,6 +151,53 @@ optional arguments:
   -d DOWNLOAD_STANDARD_NAMES, --download-standard-names DOWNLOAD_STANDARD_NAMES
                         Specify a version of the cf standard name table to
                         download as packaged version
+(cchecker) [badams@localhost]~/devel/compliance-checker% python cchecker.py --help
```

Comment:
This block below seems to be redundant to the section above. Can we combine the 2 and drop the ```[badams@localhost]~```

Reply:
Common approach would be to use `$ python cchecker.py --help`

Reply:
@benjwadams I have a task to update the README so I'll merge so I can build off this and I'll fix the couple items left here.

Reply:
Oops, sorry.  Hasty copypasta